src/routes/(console)/project-[region]-[project]/functions/function-[function]/deployment-[deployment]/header.svelte

```diff
@@ -3,17 +3,15 @@
     import { page } from '$app/state';
     import { Id } from '$lib/components';
     import { Cover, CoverTitle } from '$lib/layout';
+    import { func } from '../store';
     import { deployment } from './store';
-
-    $: projectId = page.params.project;
-    $: functionId = page.params.function;
 </script>
 
 <Cover>
     <svelte:fragment slot="header">
         <CoverTitle
-            href={`${base}/project-${page.params.region}-${projectId}/functions/function-${functionId}`}>
-            Deployment
+            href={`${base}/project-${page.params.region}-${page.params.project}/functions/function-${page.params.function}`}>
+            {$func.name}'s deployment
```

Comment:
what if the name is looong?

Reply:
but that would also truncate the text `deployment` as well right? 👀
Shouldn't the deployment page's title should mention deployment?
```
LoremIpsumLongFunctio's deployment > LoremIpsum...
```
isn't that same as function's title then?